lib/ASRouterTriggerListeners.jsm

```diff
@@ -429,14 +487,7 @@ this.ASRouterTriggerListeners = new Map([
             this,
             "SiteProtection:ContentBlockingEvent"
           );
-
-          for (let win of Services.wm.getEnumerator("navigator:browser")) {
-            if (isPrivateWindow(win)) {
-              continue;
-            }
-            win.gBrowser.removeTabsProgressListener(this);
-          }
-
+          EveryWindow.unregisterCallback(this.id);
```

Comment:
This fixes an issue from my recent `EveryWindow` refactoring, the mochitest associated with this trigger would fail with `--verify`.